matlab/src/cpp/arrow/matlab/tabular/proxy/record_batch.cc

```diff
@@ -18,17 +18,42 @@
 #include "libmexclass/proxy/ProxyManager.h"
 
 #include "arrow/matlab/array/proxy/array.h"
+#include "arrow/matlab/array/proxy/wrap.h"
+
 #include "arrow/matlab/error/error.h"
 #include "arrow/matlab/tabular/proxy/record_batch.h"
 #include "arrow/type.h"
 #include "arrow/util/utf8.h"
 
+#include "libmexclass/proxy/ProxyManager.h"
+#include "libmexclass/error/Error.h"
+
+#include <sstream>
+
 namespace arrow::matlab::tabular::proxy {
 
+    namespace {
+        libmexclass::error::Error make_empty_record_batch_error() {
```

Comment:
FYI - I used lowerCamelCase here: https://github.com/apache/arrow/pull/37013/files#diff-edaf8f57d4e37cd6648979e4cb029e5a1fdb107861fd3dd1f7faa6fc52efb1caR33. Maybe we should do the same here for consistency.

Reply:
That's a good point. I'll change the function names.